src/rules/no-focused-tests.ts

```diff
@@ -5,16 +5,25 @@ import {
 import {
   DescribeAlias,
   TestCaseName,
+  TestCaseProperty,
   createRule,
+  getNodeName,
   isSupportedAccessor,
 } from './utils';
 
+const validTestCaseNames = [TestCaseName.test, TestCaseName.it];
+
 const testFunctions = new Set<string>([
   DescribeAlias.describe,
-  TestCaseName.test,
-  TestCaseName.it,
+  ...validTestCaseNames,
 ]);
 
+const isConcurrentExpression = (expression: TSESTree.Node) =>
+  getNodeName(expression) &&
+  new RegExp(
```

Comment:
I'm not a big fan of this, as it's hard to understand what the logic will be at runtime - instead, it would be better to use standard conditional checks against the AST structure, like we do with `isDescribeEach` & co.